batchlib/workflows/cell_analysis.py

```diff
@@ -133,6 +134,8 @@ def core_workflow_tasks(config, name, feature_identifier):
     (nuc_seg_in_key, serum_seg_in_key,
      marker_ana_in_key, serum_ana_in_keys) = get_input_keys(config, serum_in_keys)
 
+    seg_key_for_infected_classification = config.seg_key + '_for_infected_classification'
```

Comment:
Does this change the table keys that are written by `FindInfectedCells`? I hope not ...

Reply:
Yes it does and it is a mess. 

I would have cleaned this up by getting rid of all the lines like `feature_table_out_key = cell_seg_key + '/' + marker_key` and just directly specifying the out path to save results at for all the jobs. This was convenient in the beginning but now it is mainly confusing..

Or, let each class figure out this path (similar to now), but return it after construction to be passed as input to the next job (this would of cause not be possible with the current `job_list` structure; one would have to initialize the jobs one by one).

Reply:
Yeah, I think in the end it would be better if each class just gets some representation of the classes it expects as requirements + each class defining input / output.
But that's too large of a refactor right now; I think I have fixed it for now.